python/cudf/cudf/tests/test_groupby.py

```diff
@@ -445,20 +445,6 @@ def test_groupby_external_series(series):
     assert_eq(pxx, gxx)
 
 
-@pytest.mark.xfail(
-    raises=NotImplementedError,
-    reason="cuDF does not support arbitrary series "
-    "index lengths for groupby",
-)
-@pytest.mark.parametrize("series", [[0, 1], [1, 1, 1, 1]])
```

Comment:
Did this test not pass? Just curious why we're deleting it

Reply:
It was expecting a exception, but now we will not come across this issue as we support arbitrary length.

Reply:
Now that it's implemented, you should just remove the `xfail` and keep this test.